build.ps1

```diff
@@ -444,6 +451,80 @@ function DownloadCefBinaryAndUnzip()
   }  
 }
 
+function CopyFromLocalCefBuild()
+{
+  # Example file names from cefsource build:
+  # 32-bit: cef_binary_3.2924.1538.gbfdeccd_windows32.tar.bz2
+  # 64-bit: cef_binary_3.2924.1538.gbfdeccd_windows64.tar.bz2
+
+  Write-Host $CefVersion
+
+  $Cef32FileName = "cef_binary_$($CefVersion)_windows32.tar.bz2"
+  $Cef64FileName = "cef_binary_$($CefVersion)_windows64.tar.bz2"
+  
+  set-alias sz "$env:ProgramFiles\7-Zip\7z.exe"
+  
+  if ([System.IO.Path]::IsPathRooted($CefBinaryDir))
+  {
+    $CefBuildDir = $CefBinaryDir
+  }
+  else
+  {
+    $CefBuildDir = Join-Path $WorkingDir "$CefBinaryDir/"
+  }
+
+  $LocalFile = Join-Path $WorkingDir $Cef32FileName
+  
+  if(-not (Test-Path $LocalFile))
+  {
+	Write-Diagnostic "Copy $Cef32FileName (approx 200mb)"
+    Copy-Item ($CefBuildDir+$Cef32FileName) $LocalFile
+	Write-Diagnostic "Copy of $Cef32FileName complete"
+  }
+
+  if(-not (Test-Path (Join-Path $Cef32 '\include\cef_version.h')))
+  {
+	# Extract bzip file
+   	sz e $LocalFile
+	# Extract tar file
+	$TarFile = ($LocalFile).Substring(0, $LocalFile.length - 4)
+	sz x $TarFile
+	#Sleep for a short period to allow 7z to release it's file handles
+	sleep -m 2000
+	#Remove tar file
+	Remove-Item $TarFile
+    $Folder = Join-Path $WorkingDir ($Cef32FileName.Substring(0, $Cef32FileName.length - 8))
+    Move-Item ($Folder + '\*') $Cef32 -force
+    Remove-Item $Folder
+  }
+  
+  $LocalFile = Join-Path $WorkingDir $Cef64FileName
+  
+  if(-not (Test-Path $LocalFile))
+  {
+	
+	Write-Diagnostic "Copy $Cef64FileName (approx 200mb)"
+    Copy-Item ($CefBuildDir+$Cef64FileName) $LocalFile;
+	Write-Diagnostic "Copy of $Cef64FileName complete"
+  }
+
+  if(-not (Test-Path (Join-Path $Cef64 '\include\cef_version.h')))
+  {
+	# Extract bzip file
+	sz e $LocalFile
+	# Extract tar file
+	$TarFile = ($LocalFile).Substring(0, $LocalFile.length - 4)
+	sz x $TarFile
+	#Sleep for a short period to allow 7z to release it's file handles
+	sleep -m 2000
+	#Remove tar file
+	Remove-Item $TarFile
+    $Folder = Join-Path $WorkingDir ($Cef64FileName.Substring(0, $Cef64FileName.length - 8))
```

Comment:
Indentation is out. Assuming you copied this and the original indentation is out and you have time please fix that also.